### PR TITLE
List Interactives endpoint

### PR DESCRIPTION
**Pull request by nimitsarup**
### What

Describe what you have changed and why.

### How to review

Describe the steps required to test the changes.

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: api/interactives.go [hunk 1]
```diff
@@ -194,8 +194,30 @@ func (api *API) UpdateInteractiveHandler(w http.ResponseWriter, req *http.Reques
 
 }
 
-func (api *API) ListInteractivessHandler(w http.ResponseWriter, req *http.Request) {
+func (api *API) ListInteractivesHandler(w http.ResponseWriter, req *http.Request, limit int, offset int) (interface{}, int, error) {
```
Comment (mryan321) [inline api/interactives.go:197]: i think better to abstract limt & offset into struct?

Comment (mryan321) [reply api/interactives.go:197]: something like https://github.com/ONSdigital/dp-census-alpha-search-api/blob/9cc640957c2096bf3d95547a4be2fd8cb1d19a41/models/pagination.go#L11 - bit cleaner imo but upto you :) prob not with that name though

**Review by mryan321 — APPROVED** [review]
lgtm - one minor suggestion but your call 👍

File: api/interactives.go (hunk 1 above)
Comment (nimitsarup) [reply api/interactives.go:197]: Agreed and it be more easy on the eyes too, point noted .. will do that in a future checkin, thanks

**Pull request merged**